hack/get_information_helper.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
```

Comment:
`get` makes it sound like an action will be performed when executing this file.
how about `project_info.sh`?

Reply:
done